tensornetwork/block_sparse/charge.py

```diff
@@ -528,16 +530,6 @@ def isin(self, target_charges: Union[np.ndarray, "BaseCharge"]) -> np.ndarray:
 
 class U1Charge(BaseCharge):
   """Charge Class for the U1 symmetry group."""
-  def __init__(self,
```

Comment:
Why are we removing the init function here and not for the Z2 and Modular charges?

Reply:
U1Charge doesn't perform any checks currently, the others do.

Reply:
Oh I see, you're defaulting to `BaseCharge`'s init, got it.